src/Uno.UI.Runtime.Skia.Gtk/GTK/UnoDrawingArea.cs

```diff
@@ -9,14 +9,31 @@ internal class UnoDrawingArea : Gtk.DrawingArea
 		private SKBitmap bitmap;
 		private int renderCount;
 		private int InvalidateRenderCount;
+		private double _dpi;
```

Comment:
Codacy found an issue: [Remove the field '_dpi' and declare it as a local variable in the relevant methods.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5924508)